torchchat/export.py

```diff
@@ -28,6 +28,31 @@
 default_device = "cpu"
 
 
+"""
+Export Snapshot
+"""
+
+
+def export_snapshot(
+    model: nn.Module,
+    device: Optional[str] = None,
```

Comment:
unused arg?

Reply:
It's a cut and paste thing, and wanted to keep args consistent.  Mind you, we could put the device in the file or some other such, and check on reload that it's the same.  (I think MPS/CPU are sorta fungible, which might help on Macs with quantization when you run out of kernel memory to quantize large models.  CPU could use paging to ssd for that. eg discussion on #1483)